### PR TITLE
Added repository-progress-view on startup and fixes description issues

### LINKED ISSUE #96: 'null' appears in the tile when there is no description
screen shot 2017-11-02 at 1 06 42 pm

### PR DESCRIPTION
**Pull request by Nitair**
## Description
- Changed commit #44 which only includes progress (and changed style slightly)
- On startup (nearly everytime on F5) showed previously just "0" which changes to "X %" on load. Now it shows "0 %" at the start
- Fixed some format (theres a lot more to fix) to make them easier to find / edit
- Also checked mobile-view and it does fit well
- This change does not decrease performance  (or its not noticeable)

## Other changes
- Added fix for description "null" if no description is given on a repository (closes #96)

### CODE REVIEW

**Comment by Nitair** [comment]
Meh, I should do my change in 

function repoDescription(repo) {
        desc = repo.description;
        return desc;
}

I will add another change for that. But for now it works.

File: index.html [hunk 1]
```diff
@@ -117,14 +121,21 @@
...
           var $item = $("<div>").addClass("card pin " + (repo.language || '').toLowerCase() + " "+repo.name.toLowerCase());
           var $link = $("<a>").attr("href", repoUrl(repo)).appendTo($item);
           $link.append($("<h4>").text(repo.name));
           $link.append($("<h5>").text((repo.language != null) ? repo.language : ""));
-          $item.append($("<p>").text(desc));
+          if(repo.description) {
+              $item.append($("<p>").text(desc));
+          }
+          else {           
+              $item.append($("<p>").text('No description given.'));
```
Comment (stevemart) [inline index.html]: I was thinking of just not showing anything here instead, thoughts on that?

Comment (Nitair) [reply index.html]: Sure.

**Comment by stevemart** [comment]
@Nitair hmm, i was notified that the page isn't building:

```
The page build failed for the `master` branch with the following error:
unable to build page. Please try again later.
For information on troubleshooting Jekyll see:
  https://help.github.com/articles/troubleshooting-jekyll-builds
```

When I pull down master, run `$ jekyll serve`, and load it in a browser, it just fine, any ideas?

**Comment by Nitair** [comment]
@stevemart It seems its a general issue from GitHub right now. I created a new repo with just index.html with content <h1>Hello World</h1> and got 

>  Your site is having problems building: unable to build page. Please try again later.

**Comment by stevemart** [comment]
@Nitair 👍 awesome

**Comment by Nitair** [comment]
@stevemart 
image

**Pull request merged**